### PR TITLE
Added url2domain.py file. Fixes issue #32

### PR DESCRIPTION
**Pull request by Yh010**
The main script needs to be modified to utilize url2domain.py file

### CODE REVIEW

**Comment by romain-fontugne** [comment]
The url2domain file looks fine but you are also adding 23 unecessary files ./history/* ?

**Comment by Yh010** [comment]
Actually since .history/ wasnt included in the gitignore file, all the history was also getting committed. changed it now..pls have a look

**Comment by romain-fontugne** [comment]
You are still pushing two history files.

**Comment by Yh010** [comment]
@romain-fontugne removed the history files now.

File: iyp/post/url2domain.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        url_id = self.iyp.batch_get_nodes('URL', 'url')
+
+        # Compute links
+        links = []
+        for url, url_qid in url_id.items():
+            # Extract domain name from URL
+            domain = tldextract.extract(url).registered_domain
+
+            # Get DomainName node for the domain
+            domain_qid = self.iyp.get_node_id('DomainName', 'name', domain)
```
Comment (romain-fontugne) [inline iyp/post/url2domain.py:20]: this method doesn't exist :D

Comment (romain-fontugne) [inline iyp/post/url2domain.py:20]: I guess you want mean batch_get_nodes, but then you shouldn't call it in the for loop. You can call it only once

**Review by romain-fontugne — CHANGES_REQUESTED** [review]

**Review by romain-fontugne — APPROVED** [review]
thanks, looks good to me

**Pull request merged**